api/v1/flow/flow.proto

```diff
@@ -375,6 +375,10 @@ message FlowFilter {
     repeated string http_method = 21;
     // http_path is a list of regular expressions to filter on the HTTP path.
     repeated string http_path = 22;
+
+    // node_names is a list of patterns to filter on the node name, e.g. (e.g.
+    // "k8s*", "test-cluster/*.domain.com", "cluster-name/**").
+    repeated string node_names = 23;
```

Comment:
There's an incoming change claiming the same proto field here: https://github.com/cilium/cilium/pull/13826/files#diff-74cced194f034ecadb39f852432c1822b6ef022065df0cb0d2ff6c718b339d45R380.

One of these PRs will require a rebase. cc @nyrahul

Reply:
Thanks for the heads-up. I'm happy to rebase this one if #13826 lands first. I'm pretty sure the ProtoBuf compiler will print an error if we accidentally use the same field number twice.